docs/frontend/translation.rst

```diff
@@ -52,15 +60,24 @@ can use the following command::
    edge-cases.
 
 
-Translation
------------
+3. Translation with Transifex
+-----------------------------
+
+We generally use `transifex`_ for translation. Note that new strings
+in the code are not automatically displayed in transifex, and
+updates in transifex are not instantly reflected in either the code
+or any running platform. 
+
+In order to push changes from the foo project code to transifex,
+the transifex-client can be used like this::
+
+   $ cd src/foo/
+   $ tx push -a
```

Comment:
If you are not careful, you might end up overwriting new translations on transifex. My personal workflow is to pull first, manually review the changes and push afterwards. We should at lease add a warning.